pkg/distro/fedora/package_sets.go

```diff
@@ -188,6 +188,101 @@ func iotCommitPackageSet(t *imageType) rpmmd.PackageSet {
 	return ps
 }
 
+func bootcContainerPackageSet(t *imageType) rpmmd.PackageSet {
+	// Replicating package selection from centos-bootc:
+	// https://github.com/CentOS/centos-bootc/
```

Comment:
I'd really like to not fork this...how about instead auto-generating this Go code from that as Source Of Truth?

Or vendoring it as a git submodule...

Reply:
Could read in the yaml files and autogenerate, keeping the commit ID pinned so it updates manually (a sort of manual submodule).